public/css/app.css

```diff
@@ -0,0 +1,160 @@
+*{
+    --dark:#0d0d0d;
+    --dark-gray:#404040;
+    --tertiary:#BABF24;
+    --secondary: #13F2A1;
+    --primary: #6DF2CC;
+    --gray: #c0c0c0;
+    --white: #f2f2f2;
+    --danger: #F26D6D;
+}
+
+@font-face {  
+    font-family: "Sofia Sans", sans-serif;
+    src: url('/fonts/SofiaSans-Regular.ttf');
+    font-weight: normal;  
+    font-style: normal;
+}
+    
+
+
+body{
+    font-family: "Sofia Sans", sans-serif;
+    background-color: var(--dark);
+    color: var(--white);
+}
+
+.text-effect{
+    border: 1px solid;
+    text-shadow: -3px 3px var(--primary), -6px 6px var(--secondary), -9px 9px var(--tertiary);
+}
+
+.input-login{
+    background-color: var(--white);
+    color: var(--dark);
+    border: 1px solid var(--primary);
+    border-radius: 10px;
+    width: 600px;
+    height: 45px;
+}
+
+.input-nav-small{
+    background-color: var(--white);
+    color: var(--dark);
+    border: 1px solid var(--gray);
+    border-radius: 20px;
+    background-image: url(https://cdn2.hubspot.net/hubfs/4004166/bioticresearch_website_assets/images/search_icon.png);
+    background-repeat: no-repeat;
+    background-position: right center;
+    width: 340px;
+    height: 25px;
+}
```

Comment:
acho que tu poderia deixar só a classe do input feita, não precisa adicionar o ícone de busca, deixe isso para quando for fazer o componente, pois provavelmente esse ícone terá que ser clicável